src/components/CheckIn/CheckInItems.js

```diff
@@ -0,0 +1,218 @@
+    const searchText = get(event, 'target.value', '').toLocaleLowerCase();
+
+    this.setState({ searchText });
+  }
+
+  // addPieceModalOpen = () => {
+  //   this.setState({ addPieceModalOpened: true });
+  // }
+
+  // addPieceModalClose = () => {
+  //   this.setState({ addPieceModalOpened: false });
+  // }
+
+  // addPieceModalSave = values => {
+  //   const { mutator } = this.props;
+
+  //   mutator.ORDER_PIECES.POST(values)
+  //     .then(() => this.callout.current.sendCallout({
+  //       type: 'success',
+  //       message: <FormattedMessage id="ui-orders.checkIn.addPiece.success" />,
+  //     }))
+  //     .catch(() => this.callout.current.sendCallout({
+  //       type: 'error',
+  //       message: <FormattedMessage id="ui-orders.checkIn.addPiece.error" />,
+  //     }))
+  //     .then(this.fetchItems);
+  // }
+
+  render() {
+    const { searchText } = this.state;
+    // const { match: { params: { lineId } } } = this.props;
+    // const initialValuesPiece = {
+    //   poLineId: lineId,
+    // };
+    const items = this.getItems();
+    const resultsFormatter = {
+      'isChecked': piece => (
+        <Checkbox
+          checked={piece.isChecked}
+          type="checkbox"
+          onChange={() => this.togglePiece(piece)}
+        />
+      ),
+      'title': piece => get(piece, 'title', ''),
+      'piece': piece => piece.caption,
+      'supplement': piece => (
+        <Checkbox
+          checked={piece.supplement}
+          disabled
+          type="checkbox"
+        />
+      ),
+      'poLineNumber': piece => piece.poLineNumber,
+      'comment': piece => piece.comment,
+      'pieceStatus': piece => piece.receivingStatus,
+    };
+
+    return (
+      <div data-test-check-in-items>
+        <Row end="xs">
+          <Col data-test-check-in-items-search>
+            <SearchField
+              marginBottom0
+              onChange={this.changeSearchText}
+              onClear={this.changeSearchText}
+              value={searchText}
+            />
+          </Col>
+          <Col>
+            <Button
+              buttonStyle="default"
+              data-test-check-in-items-add-piece-button
+              // onClick={this.addPieceModalOpen}
+            >
+              <FormattedMessage id="ui-orders.checkIn.buttons.addPiece" />
+            </Button>
+          </Col>
+          <Col>
+            <Button
+              buttonStyle="primary"
+              data-test-check-in-items-check-in-button
+            >
+              <FormattedMessage id="ui-orders.checkIn.buttons.checkIn" />
+            </Button>
+          </Col>
+        </Row>
+        <MultiColumnList
+          contentData={items}
+          formatter={resultsFormatter}
+          visibleColumns={['isChecked', 'title', 'piece', 'supplement', 'poLineNumber', 'comment', 'pieceStatus']}
+          columnMapping={{
+            isChecked: <Checkbox type="checkbox" checked={this.state.isAllChecked} onChange={() => this.toggleAll()} />,
```

Comment:
long line

it's better to break line for each prop

Reply:
fixed